### PR TITLE
Additional refinement of tqdm implementation

### PR DESCRIPTION
**Pull request by conradstorz**
I love what you have done with tqdm! I was however getting ugly progressbar display on my console. Using this option for Loguru fixes that.
I was also not getting enough progress reporting from "find_duplicates()". I get a rather long delay a that point due to my testing repeatedly from the same input file to the same output directory thus generating lots of duplicates. "Step1" and "Step2" each take considerable time.

### CODE REVIEW

File: google_photos_takeout_helper/__main__.py [hunk 1]
```diff
@@ -254,7 +253,8 @@ def find_duplicates(path: Path, filter_fun=lambda file: True):
                 files_by_size[file_size].append(file)
 
         # For all files with the same file size, get their hash on the first 1024 bytes
-        for file_size, files in files_by_size.items():
+        logger.info('Step 1...')
+        for file_size, files in _tqdm(files_by_size.items()):
```
Comment (TheLastGimbus) [inline google_photos_takeout_helper/__main__.py:257]: The "step 1" is <0 seconds for me, so I left it - how long is it for you? I would prefer to leave it as-is...

File: google_photos_takeout_helper/__main__.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
-import sys
-
+from tqdm import tqdm as _tqdm
 from loguru import logger
 
 logger.remove()  # removes the default console logger provided by Loguru.
 # I find it to be too noisy with details more appropriate for file logging.
 # INFO and messages of higher priority only shown on the console.
-logger.add(sys.stdout, format="{message}", level="INFO")
+logger.add(lambda msg: _tqdm.write(msg, end=""), format="{message}", level="INFO")
```
Comment (TheLastGimbus) [inline google_photos_takeout_helper/__main__.py:7]: Oh, it prints out logs without leaving dead bars after them... nice :sparkles: !

**Review by TheLastGimbus — APPROVED** [review]

File: google_photos_takeout_helper/__main__.py (hunk 1 above)
Comment (conradstorz) [reply google_photos_takeout_helper/__main__.py:257]: Step 1 is 27 seconds for me on a folder with ~800 duplicates. Step 2 was 50 seconds. I included the output from 2 runs below. The first is output into an empty directory with few if any duplicates and the step times are <1 second. The second run had 783 possible duplicate files to examine and required 77 seconds total to process. This is why I added the progress indication so the script would not apper to be frozen.

tldr;
My stats for the first pass over ~800 files
Copying all files to one folder...
(If you want, you can get them organized in folders based on year and month. Run with --divide-to-dates to do this)
100%|██████████████████████████████████████████████████████████████████████████████████████████████| 792/792 [01:22<00:00,  9.61it/s] 
=====================
=====================
Finding duplicates...
Step 1...
100%|████████████████████████████████████████████████████████████████████████████████████████████| 783/783 [00:00<00:00, 2365.13it/s]
Step 2...
100%|████████████████████████████████████████████████████████████████████████████████████████████████| 17/17 [00:00<00:00, 25.53it/s] 
Removing duplicates...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<?, ?it/s] 
==================

Then the second pass over same files into already populated output directory

=====================
Copying all files to one folder...
(If you want, you can get them organized in folders based on year and month. Run with --divide-to-dates to do this)
100%|██████████████████████████████████████████████████████████████████████████████████████████████| 792/792 [01:22<00:00,  9.60it/s] 
=====================
=====================
Finding duplicates...
Step 1...
100%|██████████████████████████████████████████████████████████████████████████████████████████████| 783/783 [00:27<00:00, 28.27it/s]
Step 2...
100%|██████████████████████████████████████████████████████████████████████████████████████████████| 791/791 [00:50<00:00, 15.68it/s] 
Removing duplicates...
100%|████████████████████████████████████████████████████████████████████████████████████████████| 791/791 [00:00<00:00, 3682.25it/s] 
=====================

File: google_photos_takeout_helper/__main__.py (hunk 2 above)
Comment (conradstorz) [reply google_photos_takeout_helper/__main__.py:7]: Yes!

**Comment by TheLastGimbus** [comment]
Thank you, I wouldn't have enough curage to fix this myself!

**Comment by conradstorz** [comment]
You are welcome.
